src/script.cpp

```diff
@@ -1500,11 +1500,27 @@ bool is_stealth_info_type(const operation_stack& ops)
     return ops.size() == 2 &&
         ops[0].code == opcode::return_ &&
         ops[1].code == opcode::special &&
-        ops[1].data.size() >= hash_size;
+        ops[1].data.size() <= 40;
```

Comment:
Okay, I see. So, should I implement a separate 'is_null_data_type' method for standard NULL_DATA type outputs? Because, as I understand, the stealth transaction format is a feature of libbitcoin and the original bitcoin-core has no idea about it. Am I right? Even if it's not, blockchain already contains many NULL_DATA outputs (where data < 32 bytes) and they must be handled properly, because they're standard.
But in case of separate is_null_data_type and is_stealth_info_type there will be a conflict because stealth_info_type is a subset of null_data_type.